packages/assetpack/src/core/Asset.ts

```diff
@@ -288,5 +290,24 @@ export class Asset
 
         return res;
     }
+
+    toCacheData(saveHash: boolean): CachedAsset
```

Comment:
small, but I liked that the Assets knows nothing of the cache :)

I do like that its a separate function though - could we move it to the AssetCache

Reply:
done